exercises/forth/example.go

```diff
@@ -0,0 +1,243 @@
+// Package forth implements a tiny subset of the Forth language.
+package forth
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+type operatorFn func(stack *[]int) error
+
+type operatorId byte
+
+const (
+	addOp operatorId = iota
+	subOp
+	mulOp
+	divOp
+	dropOp
+	dupOp
+	swapOp
+	overOp
+	constOp
+	userDefOp
+	endDefOp
+)
+
+type operatorTyp struct {
+	fn operatorFn
+	id operatorId
+}
+
+func Forth(input []string) (result []int, err error) {
+	if len(input) == 0 {
+		return result, nil
+	}
+
+	stack := make([]int, 0, 8)
```

Comment:
I want to populate he slice with _some_ capacity, and simply picked 8.  Can add a comment.